updates/35/asset_collections_mappings_updates.sql

```diff
@@ -679,4 +679,6 @@ INSERT INTO multiasset_mappings(collection_id, asset) VALUES (464, 'eip155:42161
 INSERT INTO multiasset_mappings(collection_id, asset) VALUES (464, 'eip155:8453/erc20:0x9d0E8f5b25384C7310CB8C6aE32C8fbeb645d083');
 *
 INSERT INTO multiasset_mappings(collection_id, asset) VALUES (246, 'BONK');
+*
+DELETE FROM multiasset_mappings WHERE asset='eip155:1/erc20:0xFE67A4450907459c3e1FFf623aA927dD4e28c67a' AND collection_id='247'
```

Comment:
Do we have the new one? https://www.coingecko.com/en/coins/everclear-2 Also the collection has other assets. Shouldn't you delete also all of them? I would rename the old collection to Everclear (old) and make sure that the new token is in there

Reply:
> Do we have the new one? https://www.coingecko.com/en/coins/everclear-2 

yes, we do.

>  Shouldn't you delete also all of them? 

no, the other chains use an upgradeable proxy so they're updated to Everclear(name) & CLEAR(symbol). The only exception to this is ethereum token.

that's why i removed it from the collection.

so i updated the collection info's name, symbol and main asset